### PR TITLE
✨ Support amp-fx=float-in-* in viewer

### PR DESCRIPTION
**Pull request by alanorozco**
scroll-toggle-chingon

Reverting the revert.

@jridgewell would love to continue the conversation from #21279.

### CODE REVIEW

**Comment by aghassemi** [comment]
This is just revert of revert correct? Since initial revert turned out not to be necessary, let's try to get this in for next weeks Canary.

**Comment by aghassemi** [comment]
@alanorozco @jridgewell ping.

**Comment by alanorozco** [comment]
@jridgewell wanted to make some additional comments. I would personally prefer a rollback of the rollback.

**Comment by jridgewell** [comment]
I'll review this later today.

**Comment by aghassemi** [comment]
Let's aim to have this working by end of April plz?

**Comment by alanorozco** [comment]
ping @jridgewell ?

File: src/service/fixed-layer.js [hunk 1]
```diff
@@ -306,37 +321,23 @@ export class FixedLayer {
...
-      this.elements_.forEach(e => {
-        if (e.fixedNow && e.top) {
-          setStyles(e.element, {
-            transform: '',
-            transition: '',
-          });
-        }
-      });
-    }
+  transformMutate(opt_transform) {
```
Comment (jridgewell) [inline src/service/fixed-layer.js]: I still think this should be moved out of fixed-layer and into viewport-impl. All that's needed is to return an array of currently fixed-top elements, and iterate of that list. FixedLayer itself has no concern with `transform` styles, there's really no reason for it to live here.

File: src/layout.js [hunk 2]
```diff
@@ -25,10 +25,29 @@ import {isFiniteNumber} from './types';
 import {setStyle, setStyles, toggle} from './style';
 import {startsWith} from './string';
 
+/**
```
Comment (jridgewell) [inline src/layout.js]: Can we undo all of these unrelated file changes?

**Comment by aghassemi** [comment]
ping @alanorozco

**Comment by mrjoro** [comment]
Is this PR still active?

**Comment by danielasy** [comment]
Hello! Any news on this issue?

**Comment by kristoferbaxter** [comment]
Closing inactive PRs. @alanorozco can reopen if there are new thoughts in the space.
